index.html

```diff
@@ -1,16 +1,91 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Oksana's Cake webpage</title>
     <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;1,300;1,400&family=Lobster&display=swap"
+      rel="stylesheet"
+    />
+  </head>
 
-<body>
-    <!-- Add your markup here -->
-</body>
+  <body>
+    <header>
+      <div class="banner">
+        <img class="logo" src="./images/logo-cupcake.svg" alt="company logo" />
+        <p class="header-shop-name">Pastryfied cake shop</p>
+        <img src="./images/menu-hamburger.svg" alt="hamburger icon" />
+      </div>
+      <h3 class="header-text">The best cakes in town delivered to your door</h3>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li><a href="#">home</a></li>
+          <li><a href="#">cakes</a></li>
+          <li><a href="#">order in</a></li>
+          <li><a href="#">lessons</a></li>
+          <li><a href="#">about</a></li>
+        </ul>
+      </nav>
+    </header>
 
-</html>
\ No newline at end of file
+    <main>
+      <section class="hero">
+        <article class="hero-text">
+          <h1>Welcome to London's best cake shop</h1>
+          <p>
+            We bake our cupcakes, cakes and all kind of desserts every morning
+            using the worlds finest ingredients. We deliver all across London so
+            whether you need a rainbow cake, corporate branded cupcakes or just
+            an epic special occasion or birthday cake we've got you. This is THE
+            place to order cupcakes online for London delivery.
+          </p>
+        </article>
+        <img
+          class="hero-img"
+          src="./images/cake.jpg"
+          alt="picture of the cake"
+        />
+      </section>
+
+      <section class="cards">
+        <div class="cards-container cards-first">
+          <img src="./images/dessert1.jpg" alt="" />
```

Comment:
Remember to include `alt` text here for your image. We want to make our webpages accessible to everyone, and that alt text is really important to someone who is visually impaired, for example